components/api/src/test/java/com/hotels/styx/api/io/FileResourceIndexTest.java

```diff
@@ -27,14 +27,15 @@
 import static org.hamcrest.Matchers.contains;
 
 public class FileResourceIndexTest {
-    Path PLUGINS_FIXTURE_PATH = ResourcePaths.fixturesHome(FileResourceIndexTest.class, "/plugins");
-    FileResourceIndex resourceIndex = new FileResourceIndex();
+    private Path PLUGINS_FIXTURE_PATH = ResourcePaths.fixturesHome(FileResourceIndexTest.class, "/plugins");
+    private FileResourceIndex resourceIndex = new FileResourceIndex();
+    private String EXISTING_PLUGIN_RELPATH = "oneplugin/url-rewrite-1.0-SNAPSHOT.jar";
 
     @Test
     public void listsResourcesFromFileSystemDirectory() {
         Iterable<Resource> jars = resourceIndex.list(PLUGINS_FIXTURE_PATH.toString(), ".jar");
 
-        assertThat(jars, contains(resourceWithPath("oneplugin/url-rewrite-1.0-SNAPSHOT.jar")));
+        assertThat(jars, contains(resourceWithPath(EXISTING_PLUGIN_RELPATH)));
```

Comment:
We follow a principle of leaving literal strings in tests where possible so that readers do not need to jump around the file to understand the test.